CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing to CICE
+
+Welcome to the CICE Consortium ! First, thanks a lot for considering to contribute to CICE.
+
+There are many ways to contribute, and we appreciate your efforts. For an introductory tour, head over to our [Contribution guide][contributing] and take a look at the [Resources page], our go-to index with links to code, documentation, testing information, guidelines, and more.
+
+[contributing]: https://github.com/CICE-Consortium/About-Us/wiki/Contributing
+[resources]: https://github.com/CICE-Consortium/About-Us/wiki/Resource-Index
+
+## Getting help
+If you still need help getting started using the model afer reviewing the [model documentation][doc-running], the first point of contact with the CICE Consortium is the [Consortium Community Forum][forum]. 
+This forum is monitored by Consortium members and also opened to the whole community.
+**Please do not use our issue tracker for general support questions.**
```

Comment:
Maybe have this on its own line?

Reply:
I might even exclude this point of information.  Lets wait to see if we have problems with it before we say don't do it.  We point people to the forum early on, they should not end up on the issues page.

Reply:
I can go either way on this point.  It's not a bad idea to make it very explicit because we've had quite a number of people submitting issues unnecessarily in the past.  Of course, they might not read this page so the problem will likely not ever go away completely.

Reply:
I decided to leave the statement in place.